src/plugins/js-eval/Dockerfile

```diff
@@ -15,12 +15,17 @@ RUN node --version
 RUN npm i --prefix /run airbnb-js-shims@latest string.prototype.at@latest array.prototype.at@latest @bloomberg/record-tuple-polyfill@latest nearley@latest array.prototype.group@latest array.prototype.grouptomap@latest array.prototype.flat@latest array.prototype.flatmap@latest array.prototype.with@latest array.prototype.tosorted@latest array.prototype.toreversed@latest array.prototype.tospliced@latest set.prototype.union set.prototype.intersection set.prototype.difference set.prototype.symmetricdifference set.prototype.issubsetof set.prototype.issupersetof set.prototype.isdisjointfrom es-set es-map suppressed-error es-iterator-helpers arraybuffer.prototype.detached disposablestack
 RUN curl https://engine262.js.org/engine262/engine262.js -o /run/node_modules/engine262.js
 
-FROM debian:10-slim
-
 RUN useradd node && mkdir -p /home/node && chown -R node:node /home/node
 WORKDIR /home/node
-COPY --from=0 /run /run
-COPY --from=0 /usr/local/bin/node /usr/local/bin/node
```

Comment:
These `COPY --from=0 ..` commands were failing locally. I don't think there is a reason for them to be there as well?

Reply:
it seems like it's a permissions resetting thing. we can try it, but this might make deployment fail.